deps.edn

```diff
@@ -35,7 +34,9 @@
         org.babashka/sci                          {:mvn/version "0.9.44"}
         org.mentat/emmy-viewers                   {:mvn/version "0.3.2"}
         
-        org.scicloj/clay                          {:mvn/version "2-beta30"}}
+        org.scicloj/clay                          {:mvn/version "2-beta31.1"}
+
+        org.clojars.kjw/slf4j-simple {:mvn/version "1.5.5"}}
```

Comment:
I think slf4j-simple is wrong "here".
So everyboddy who is using "noj" as library would pull it in.

It needed to be in an alias which is only used for uberjar generation

Reply:
:clojupyter is such alias.

I am not sure neither, if adding it would harm running the uberjar as clojupyter kernel, probbaly not.

The clojupyter kernel is an "executable" as well, so should rightly have a logger implemenation.

Reply:
maybe the name of the alias is now wrong.
The uberjar has now 2 entry points :
- clojupyter kernel
- clay main/server
(+ Clojure repl)

Reply:
maybe :uber   would be better.